modules/ballerina-builtin/pom.xml

```diff
@@ -58,6 +58,11 @@
             <groupId>org.apache.ws.commons.axiom</groupId>
             <artifactId>axiom-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.ecf</groupId>
+            <artifactId>org.apache.commons.codec</artifactId>
+            <version>1.9.0.v20170208-1614</version>
```

Comment:
This is not the correct way to add a dependency version. You have to define versions in ballerina-parent (repo) pom.xml as a property.